### ISSUE TITLE
try-best-params-from-first-pass-tpe-study

### ISSUE THREAD

**Issue by david-thrower** [david-thrower/cerebros-core-algorithm-alpha#178]
{
'embedding_n': 12, 
'activation': 'gelu', 
'predecessor_level_connection_affinity_factor_first': 38.7, 
'predecessor_level_connection_affinity_factor_main': 4.6, 
'max_consecutive_lateral_connections': 32, 
'p_lateral_connection': 17.1, 
'num_lateral_connection_tries_per_unit': 1, 
'learning_rate': 0.046500145665525995, 
'epochs': 7, 
'batch_size': 22, 
'dropout': 0.6500000000000001, 
'maximum_units_per_level': 7, 
'maximum_neurons_per_unit': 7, 
'temperature': 36912 # iRoPE theta
}
